clock_exercise/ClockPage.qml

```diff
@@ -6,6 +6,7 @@ Rectangle {
     signal timerButtonClicked
     signal alarmButtonClicked
     signal clickOnTimerIcon
+    signal clickOnBellIcon
```

Comment:
Il naming che viene dato ai segnali in Qt quando si parla di eventi di click segue la regola del participo passato:
```qml
signal timerIconClicked
signal bellIconClicked
```
Tra l'altro guardando i segnali più vecchi, si vede che inizialmente stavi seguendo questa convenzione (ad esempio con `timerButtonClicked`).
Evitiamo se possibile di introdurre disallineamenti di questo genere nel naming per cui per metà si nominano cose con un pattern, per metà con un altro, perchè sul lungo periodo si rischia di generare confusione.

In questo caso si potrebbe anche valutare se nominare i segnali in base alla funzione piuttosto che all'interazione grafica (il click su questa o quell'altra icona), ma preferirei parlarne anche con Luca prima per sapere cosa ne pensi anche lui.